android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java

```diff
@@ -53,27 +55,15 @@
 
   @Override
   public void load() {
-    String clientId = getConfig().getString("androidClientId",
```

Comment:
since both [requestIdToken](https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInOptions.Builder#public-googlesigninoptions.builder-requestidtoken-string-serverclientid) and [requestServerAuthCode](https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInOptions.Builder#public-googlesigninoptions.builder-requestserverauthcode-string-serverclientid) require `serverClientId`, I removed this

Reply:
Strange it is so different from iOS, but docs are clear